OpenTabletDriver.UX/Controls/Generic/GeneratedItemList.cs

```diff
@@ -98,6 +98,10 @@ protected virtual void HandleCollectionChanged(object sender, NotifyCollectionCh
 
         protected virtual void Insert(int index)
         {
+            // Prevent an exception when a tablet is disconnected & a binding editor is closed
+            if (ItemSource is null || index >= ItemSource.Count)
```

Comment:
the latter check (`index >= ItemSource.Count`) will definitely help in avoiding some exceptions but I feel like asking for an index that is higher than the count should warn the developer instead?
```suggestion
            Debug.Assert(index >= ItemSource.Count);
            if (ItemSource is null)
```

Alternatively you can make it fall back to the max, but that may have other side effects:
```suggestion
            index = (index >= ItemSource.Count) ? ItemSource.Count - 1 : index
            if (ItemSource is null)
```

(the suggested code might not compile or may have an off-by one, please test manually first)